src/index.ts

```diff
@@ -657,6 +694,54 @@ async function mergeNeurons(sourceNeuronId: bigint, targetNeuronId: bigint) {
   ok();
 }
 
+async function registerVote(neuronId: bigint, proposalId: bigint, vote: Vote) {
```

Comment:
in the other function you use object for parameters. this was an on purpose choice to not to here and in the new functions of this PR?